setup.cfg

```diff
@@ -53,6 +53,7 @@ cli =
     sentry-sdk[fastapi]>=1.9.0
 dev =
     bandit==1.7.4
+    bcrypt==4.0.1
```

Comment:
Isn't `bcrypt` a `cli` dependency as `src/ralph/cli.py` uses it?

Reply:
Well spotted! It should be. I thought I did this. I bet it's a `git rebase` issue :roll_eyes: